.github/workflows/lerna.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
           node-version: 18
           cache: "npm"
       - name: Install deps
-        run: npm ci
+        run: npm install
```

Comment:
Please don't change this to install. We want to make sure the lock is updated.

Reply:
yes, will fix. (i'm just trying to debug the ci failure, which works fine locally, so not sure what's going on)